src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexService.java

```diff
@@ -335,6 +360,52 @@ private String indexFieldType(String exportField) {
         }
     }
 
+    /**
+     * Indexes all exported objects for this project
+     * @param options
+     * @throws IOException
+     */
+    public void indexAllFromCsv(CdmIndexOptions options) throws IOException {
+        assertObjectsExported(options);
+
+        CdmFieldInfo fieldInfo = fieldService.loadFromCsvFieldsFromProject(project);
```

Comment:
use `loadFieldsFromProject`. The only time we should need to load the fields from the CSV file is when populating the field info object for persisting purposes